pennylane_cirq/cirq_device.py

```diff
@@ -71,6 +71,8 @@ def __init__(self, wires, shots, analytic, qubits=None):
 
         self.circuit = None
 
+        device_wires = self.map_wires(self.wires)
```

Comment:
Oh, so if I understand this correctly, `self.map_wires` will in this case go from e.g. `self.wires -> ['q1', 'ancilla', 0, 1]` to `device_wires -> [0, 1, 2, 3]`. And then the list of wires can be extracted from the `Wires` object with `device_wires.labels`. Is this correct?

Reply:
Yes, perfect! Well, strictly speaking the map goes from ``self.wires -> Wires(['q1', 'ancilla', 0, 1]) to device_wires -> Wires([0, 1, 2, 3])``, and ``device_wires.labels`` is the tuple ``(0, 1, 2, 3)``...